docs/modules/layout.md

```diff
@@ -0,0 +1,20 @@
+## Project layout
```

Comment:
I have moved the layout section above modules since now we also show the layout for documentation too. It makes more sense to have it after Installation menu common to the suite